### PR TITLE
getStrings fix, setting ADB java version to 2.0

### PR DESCRIPTION
**Pull request by scottdixon**
@imurchie

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class AndroidDriver extends BaseDriver {
       }
 
       // set up an instance of ADB
-      this.adb = await ADB.createADB();
+      this.adb = await ADB.createADB({javaVersion: 2.0});
```
Comment (imurchie) [inline lib/driver.js]: Should this not be `this.opts.javaVersion`?

Comment (scottdixon) [inline lib/driver.js]: Good catch. Updated.

**Comment by imurchie** [comment]
LGTM

**Comment by jlipps** [comment]
:+1:

**Pull request merged**